app/Project.php

```diff
@@ -18,11 +18,18 @@ class Project extends Model
     /**
      *  The path to the project.
      *
+     * @param string $add The additional path for the Project url
      * @return string
      */
-    public function path()
+    public function path($add = null)
     {
-        return "/projects/{$this->id}";
+        $path = "/projects/{$this->id}";
```

Comment:
No need to check if `$add` is null. You can pass a second parameter to `trim` to trim the trailing slash if needed.

```suggestion
        return trim("/projects/{$this->id}/{$add}","/");
```